plugins/content_distribution/providers/tvinci/lib/xml/tvinci_default.xslt

```diff
@@ -452,31 +457,28 @@
                     </xsl:for-each>
                 </xsl:variable>
 
-                <xsl:variable name="suffix">
-                    <xsl:call-template name="suffix-matching-tag">
-                        <xsl:with-param name="tag" select="$tag"/>
-                    </xsl:call-template>
+                <xsl:variable name="nameSuffix">
+                    <xsl:if test="$isChild">
+                        <xsl:variable name="trailerSuffix">
+                            <xsl:value-of select="concat(' ',$CONST_TRAILER_NAME)"/>
+                        </xsl:variable>
+                        <xsl:variable name="trailerIdxSuffix">
+                            <xsl:if test="(($childIdx > 1))">
+                                <xsl:value-of select="concat(' ',$childIdx)"/>
+                            </xsl:if>
+                        </xsl:variable>
+                        <xsl:value-of select="concat($trailerSuffix, $trailerIdxSuffix)"/>
+                    </xsl:if>
                 </xsl:variable>
-
                 <xsl:variable name="typeName">
-                    <xsl:call-template name="type-name-matching-tag">
-                        <xsl:with-param name="tag" select="$tag"/>
-                        <xsl:with-param name="isChild" select="$isChild"/>
-                        <xsl:with-param name="childNumber" select="$childIdx"/>
-                    </xsl:call-template>
-                </xsl:variable>
-
-                <xsl:variable name="ppvModule">
-                    <xsl:call-template name="ppvModule-matching-tag">
-                        <xsl:with-param name="tag" select="$tag"/>
-                    </xsl:call-template>
+                    <xsl:value-of select="concat($tag/typename, $nameSuffix)" />
                 </xsl:variable>
 
                 <xsl:call-template name="create-file-element">
-                    <xsl:with-param name="cdnCode" select="concat($playManifestPrefix, $relevantEntryId, $suffix)"/>
+                    <xsl:with-param name="cdnCode" select="concat($playManifestPrefix, $relevantEntryId, '/format/',$tag/format, '/tags/', $tag/tagname, '/protocol/',$tag/protocol,'/f/a.',$tag/extension)"/>
```

Comment:
I would extract the suffix into a function as well - since this can be a place where the PS makes their changes it will be more usable if has the scope of a function